integration/test_jobs.py

```diff
@@ -277,3 +277,58 @@ def test_results_not_available_error(
     assert isinstance(execute_results[0].download_result(), BackendResult)
 
     assert isinstance(execute_results[0].download_backend_info(), BackendInfo)
+
+
+def test_submit_under_user_group(
+    _authenticated_nexus_circuit_ref: CircuitRef,
+    qa_project_name: str,
+    qa_circuit_name: str,
+) -> None:
+    """Test that a user can submit jobs under a user_group that
+    they belong to.
+
+    Requires that the test user is a member of a group called:
+    'QA_IntegrationTestGroup',
+    and not a member of a group called:
+    'made_up_group'.
```

Comment:
I guess you've created this group for the test user in QA. Do you think you could create the same group for the corresponding org in DEV? Also, what test user are we using for the qnexus integration tests?

Reply:
Currently we are using `qa_nexus_staging_pytket-nexus@mailsac.com` and target QA in integration tests. Is this the correct user to use? It doesn't exist in dev but I can add it

Reply:
I can create the necessary org and test users in DEV later, no worries :)